### DISCUSSION TITLE
How to export ERD to SVG?

### DISCUSSION THREAD

**Discussion by haisamido** [dbeaver/dbeaver#22881] [General]
How can one export an ERD to SVG?

**Comment by haisamido** [comment]
I found https://github.com/dbeaver/dbeaver/issues/649 and will try the suggestions there.

**Comment by LonwoLonwo** [comment]
Hello @haisamido 

Yes, you can export diagrams in the SVG format. And many others. You can find the Export button right behind the Pring button.

Like this:

![2024-02-21 11_33_47-DBeaver 23 3 5 - DBeaver Sample Database (SQLite)](https://github.com/dbeaver/dbeaver/assets/45152336/09a88017-d95f-4631-a848-aeca073e1d1b)

**Reply by haisamido** [reply]
@LonwoLonwo `SVG format (*.svg)` does't appear in `Version 23.3.5.202402181738` on my mac os x system

**Reply by LonwoLonwo** [reply]
Did you install the Office Plugin in DBeaver?

> This extension supports data export in Office formats (XLS).
https://github.com/dbeaver/dbeaver/wiki/Extension-SVG-Format

**Reply by haisamido** [reply]
Thank you @LonwoLonwo for your quick and very helpful replies. That worked.

**Reply by AlexanderMakarov** [reply]
https://github.com/dbeaver/dbeaver/wiki/Extension-SVG-Format link doesn't work anymore. DBeaver v25.0.0.

Could you please help to find SVG extension?
